src/utilities/gql.ts

```diff
@@ -2428,6 +2722,33 @@ const gql = {
     }
   `,
 
+  SUB_PLAN_DERIVATION_GROUP: `#graphql
+    subscription SubPlanExternalSource {
+      links: ${Queries.PLAN_DERIVATION_GROUP}(order_by: { plan_id: asc }) {
```

Comment:
Would it make sense to make this only query for the groups that are for a specific plan rather than all of them? It seems like it's being filtered down to whatever plan is being viewed in other stores and components.

Reply:
Yeah, I don't see why that would hurt. We can go ahead and implement that.

Reply:
Refactored in [refactor planDerivationGroupLinks store to only include those associated with current plan](https://github.com/NASA-AMMOS/aerie-ui/commit/739df96f191c7e1619880aa57a0d1eafa3793d4c).